Source/PartnerCenter.CustomerPortal/Resources.Designer.cs

```diff
@@ -1951,7 +1960,7 @@ public class Resources {
         }
         
         /// <summary>
-        ///   Looks up a localized string similar to Paypal live account.
+        ///   Looks up a localized string similar to Live account.
         /// </summary>
         public static string LiveAccountTypeCaption {
```

Comment:
>LiveAccountTypeCaption [](start = 29, length = 22)

Since you are updating existing strings. Please update the localization on other languages.

Reply:
Done.

---
In reply to: [183243264](https://github.com/PartnerCenterSamples/Reseller-Web-Application/pull/61#discussion_r183243264) [](ancestors = 183243264)